src/Body.cpp

```diff
@@ -234,7 +234,7 @@ void Body::UpdateFrame()
 		FrameId parent = frame->GetParent();
 		Frame *newFrame = Frame::GetFrame(parent);
 		if (newFrame) { // don't fall out of root frame
-			Output("%s leaves frame %s\n", GetLabel().c_str(), frame->GetLabel().c_str());
+			// Output("%s leaves frame %s\n", GetLabel().c_str(), frame->GetLabel().c_str());
```

Comment:
If this is not a performance issue, I'd recommend converting these calls to `Log::Verbose("{} leaves frame{}\n", GetLabel(), frame->GetLabel());`. This way they don't show up in the console, but are still logged for debugging perusal if the log system is set to allow verbose messages in the log file.

Reply:
Done!